src/frontend/reducers.js

```diff
@@ -15,4 +15,4 @@ export default req.keys().reduce((acc, key) => {
   acc[name] = req(key);
 
   return acc;
-}, { router: routerStateReducer, form: formReducer });
+}, { routing: routeReducer, form: formReducer });
```

Comment:
I would like to use default configuration, but I don't want to forget make those changes.